DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs

```diff
@@ -205,6 +205,9 @@ private async Task FetchConfigAsyncWithTask()
 
                 logger.LogError(finalError.ErrorResponse.Message);
                 dvcEventArgs.Errors.Add(finalError);
+            } catch (Exception e) {
+                logger.LogError(e.Message);
+                dvcEventArgs.Errors.Add(new DVCException(HttpStatusCode.InternalServerError, new ErrorResponse(e.Message)));
```

Comment:
is there a version of DVCException that doesn't have a "status code"? doesn't really make sense to be setting an http status code here